src/Raven.Server/Config/RavenConfiguration.cs

```diff
@@ -42,6 +42,7 @@ public class RavenConfiguration
 
         public bool Initialized { get; private set; }
 #endif
+        public LicenseConfiguration Licensing { get; }
```

Comment:
why do we need Licensing config in RVN?

Reply:
I guess for the secure package that requires a license